### ISSUE TITLE
vued3tree import crash program without warnings

### ISSUE THREAD

**Issue by Claus1** [David-Desmaisons/Vue.D3.tree#7]
### Steps to reproduce
take tree example demo
compile it with newest vue version.
See a blank page with no warnings.

### Versions
webpack 2.6.1
vue 2.4.2
vued3tree@3.4.2
vuetify 0.15.3
Firefox 55.0.3
win 7

In my program string

import {tree} from 'vued3tree'

causes a blank page with no warnings after recompiling.

Using sources in programs changes nothing. All dependencies in Tree.vue are resolved correctly.
But can not import Tree.vue or  vued3tree correctly.

**Comment by David-Desmaisons** [comment]
Hello @Claus1 , I am not sure if I understand what you mean by "crash program". Is webpack build failing? If so what are corresponding traces?

**Comment by Claus1** [comment]
Hello David! Building is ok. it does not work in browser. There are no corresponding traces.
Just "import {tree} from 'vued3tree' " in any .vue file and a program show blank page with no warnings.
With Chrome the same story but in the left bottom corner it shows '1 error' but impossible to get any additional info from there. Firefox shows just blank page and info about vue is running in dev mode.  
Loading module error but why? All submodules are  loading correctly.

**Comment by David-Desmaisons** [comment]
I guess the best is to share the repo where you have this problem so I can take a look at it

**Comment by Claus1** [comment]
Agreed. Pls install project https://github.com/vuetifyjs/webpack and run it (run dev).  in file App.vue in script section after <script> write pls
`import {tree} from 'vued3tree'`
 
After hot reload you ll get blank page.

**Comment by David-Desmaisons** [comment]
Well, I can not reproduce after installing vuetifyjs and before installing vued3tree, I got the following error when running npm run dev:

![image](https://user-images.githubusercontent.com/524617/30431091-d7dab910-9933-11e7-8546-cc6c59740501.png)

**Comment by David-Desmaisons** [comment]
Did you try with officiel vue-cli template?

**Comment by Claus1** [comment]
It is weird. I tried https://github.com/vuetifyjs/webpack from scratch - it works for me. npm, node(6.11.1 mine) may be different. 

I checked another one https://github.com/vuetifyjs/a-la-carte with the same result.
Install it pls. After change host in package.js to localhost. it is 0.0.0.0 there.
install vued3tree. run dev. add import {tree} from 'vued3tree' to App.vue script.

I think same result will be in any project with vuetify.
